lists-service/cicd/build/pipeline/build_lists_buildspec.yaml

```diff
@@ -0,0 +1,48 @@
+version: 0.2
+
+env:
+  shell: bash
+  variables:
+    JAVA_TOOL_OPTIONS: -Dhttps.protocols=TLSv1.2
+
+phases:
+  install:
+    commands:
+      - echo Installing dependencies...
+      - apt update -y
+        && cat /etc/lsb-release
+        && apt-get -q -y install openjdk-11-jdk
+        && apt-get -q -y install unzip
+        && apt-get -q -y install zip
+        && apt-get -q -y install curl
+        && curl -s "https://get.sdkman.io" | bash
+        && chmod a+x /root/.sdkman/bin/sdkman-init.sh
+        && source /root/.sdkman/bin/sdkman-init.sh
+        && apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+        && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+        && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+        && apt-get update
+        && apt-get install -y docker-ce
+        && apt-get install -y maven
+
+  build:
+    commands:
+      - echo Build started on $(date)
+      - echo Logging in to Amazon ECR...
+      - aws ecr get-login-password | docker login -u AWS --password-stdin "https://$AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com"
+      - echo Building the Docker image...
+      - docker build -t lists-service -f lists-service/src/main/docker/Dockerfile lists-service
+      - echo Tagging the Docker image...
+      - export REPO=$AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com/$LISTS_ECR_REPOSITORY_NAME
+      - echo REPO=$REPO
+      - echo tagging
+      - docker tag lists-service:latest $REPO:$CLEAN_BRANCH-$CODEBUILD_BUILD_NUMBER
+      - docker tag lists-service:latest $REPO:latest
```

Comment:
The image name and tag can be configuration values. [here's an example from userdetails](https://github.com/AtlasOfLivingAustralia/userdetails/blob/feature/containerization/cicd/package/pipeline/build_userdetails_buildspec.yaml#L35)